ledger-core/network/nds/uniproto/l2/uniserver/server.go

```diff
@@ -39,6 +39,24 @@ type ServerConfig struct {
 	RetryDelayMax      time.Duration
 }
 
+func (cfg ServerConfig) IsZero() bool {
+	switch {
+	case cfg.BindingAddress != "" || cfg.PublicAddress != "":
```

Comment:
use only one criteria - there is no need for all

Reply:
what do u mean?

Reply:
leave only the first case. The rest is excessive

Reply:
Fixed